### ISSUE TITLE
Open the source

### ISSUE THREAD

**Issue by jcwenger** [dingram/glypher#11]
I'd love to fork and help contribute.  Open it for all of us?

**Comment by peterrecore** [comment]
I got excited when I saw a github URL, then got sad when I saw the empty code repo.

**Comment by justjkk** [comment]
:+1:

**Comment by bogeyman** [comment]
+1

**Comment by fenhl** [comment]
:+1:

**Comment by dingram** [comment]
I'm hoping to be able to open the source, but there are some limitations with my employer that I'm working out.

**Comment by JDiPierro** [comment]
+1

**Comment by pholcroft** [comment]
Yes +1. I'd love to add in the new training mode suggested. I came here to suggest it myself!

**Comment by AGSPhoenix** [comment]
@dingram, any progress on this?

**Comment by fortepc** [comment]
@dingram would really love to see this app get updated with new glyphs/sequences since you seem to have moved on from the project, any chance you were able to get that limitation with your employer worked out?
